templates/components/products/carousel.html

```diff
@@ -40,5 +40,6 @@
     {{/each}}
     {{#if products.length '>' 1}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{products.length}}" class="{{#if customArrowClass}}{{customArrowClass}} {{/if}}js-product-next-arrow slick-next slick-arrow"></button>
```

Comment:
Do we need to add if arrows statement here?

Reply:
arrows in product carousel are always true, because we have this prop in data-slick object of product carousel